backend/registered_user/views.py

```diff
@@ -286,13 +286,32 @@ def resend_verification_code(request):
     return Response({'error': 'Invalid Email'}, status=status.HTTP_400_BAD_REQUEST)
 
 
+@api_view(["POST"])
+@permission_classes((AllowAny,))
+def forgot_password(request):
+    email = request.POST.get('email', None)
+    if email is None:
+        return Response({'error': 'Missing email parameter'}, status=status.HTTP_400_BAD_REQUEST)
+    registered_user = RegisteredUser.objects.filter(email=email).first()
```

Comment:
Email is stored in Django user in RegisteredUser model so the filtering should be done with user__email instead of email